style.css

```diff
@@ -22,3 +22,68 @@ body {
 }
 
 /* Add styles here! */
+
+/* Typography */
+h1 {
+  font-size: 2rem;
+  margin-block: 32px;
+}
+
+h2 {
+  font-size: 1.5rem;
+  margin-block: 24px;
+}
+
+p {
+  font-size: 1rem;
+  line-height: 1.5;
+  margin-block: 16px;
+}
+
+/* Layout */
+header {
+  padding-block: 64px;
+}
+
+main {
+  padding-block: 64px;
+  border-top: 8px solid var(--color-gray-medium);
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches and the blocking or non-blocking decorations to further communicate your intent.  -->
**💬 suggestion:** ‏You could logical properties
`border-block-start: 8px solid var(--color-gray-medium);`